docs/design-goals.md

```diff
@@ -26,7 +26,7 @@ roadmap.
 ### Storage Scalability
 
 Systems like ZooKeeper and Etcd have limitation around ~2GB (See
-[Etcd docs](https://etcd.io/docs/v3.5/dev-guide/limit/#storage-size-limit)), mainly because they are taking
+[Etcd docs](https://etcd.io/docs/v3.5/dev-guide/limit/#storage-size-limit)), mainly because they are holding data and indexes in memory.
```

Comment:
no, the missing part here was "..because they are taking periodic snapshots of the entire data set". Etcd does not keep all data in memory.

Reply:
ok fixed it